frontend/src/scenes/persons/relatedFeatureFlagsLogic.ts

```diff
@@ -57,7 +57,7 @@ export const relatedFeatureFlagsLogic = kea<relatedFeatureFlagsLogicType>([
                 loadRelatedFeatureFlags: async () => {
                     const response = await api.get(
                         `api/projects/${values.currentProjectId}/feature_flags/evaluation_reasons?${toParams({
-                            distinct_id: props.distinctId,
+                            ...(props.distinctId ? { distinct_id: props.distinctId } : {}),
```

Comment:
this API will 500 if we don't give it a distinct_id, but that won't break the page.  TBH, the distinct_id at this point should never be `null` in practice, I just wanted to make the types more accurate.